Service Connector  Hub/func.py

```diff
@@ -22,7 +22,7 @@ def process(body: dict) -> None:
     payload.update({"source": source})
     payload.update({"time": time})
     payload.update({"data": data})
-    payload.update({"ddsource": DD_SERVICE})
+    payload.update({"ddsource": DD_SOURCE})
```

Comment:
This might require update in the test function as well, right ?

Reply:
The test is already updated with this value.

Reply:
Sounds good.